Integrations/integration-ArcSightXML.yml

```diff
@@ -0,0 +1,193 @@
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Directory from which to get XML files and create incidents.
+  name: inputDirPath
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Directory to which put command XML files.
+  name: commandsDirPath
+  defaultvalue: ""
+  type: 0
+  required: true
+script:
+  script: |+
+    import glob
+    import xml.etree.ElementTree as ET
+    import lxml.etree
+    import datetime
+    import json
+    import os
+
+    SECURITY_EVENT_START_TAG = '<SecurityEvent id="%s"'
+    SECURITY_EVENT_END_TAG = '</SecurityEvent>'
+
+    LIST_CASE_ENTRIES = ['caseEvents', 'childOf', 'ownedBy']
+    TIME_CASE_ENTRIES = ['detectionTime', 'estimatedStartTime']
+
+    SYSTEM = ''
+    URI_PREFIX = ''
+    TOREPLACE = '\\"\\<\\>\\(\\)'
+
+    XML_TEMPLATE = """<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This template did not work, please make it smaller and stick to the format I gave you.